api/app/composer.json

```diff
@@ -23,6 +23,7 @@
     "require": {
         "php": "^8.1",
         "aws/aws-sdk-php": "3.299.1",
+        "cmixin/business-day": "^1.18",
```

Comment:
Why are we using an external library that doesn't pull from the GOV Json list? The referenced library doesn't include all bank holidays either, see - https://github.com/kylekatarnls/business-day/blob/dd2e46ec7e036ccd9b818c49183dcbf0b2a43f10/src/Cmixin/Holidays/gb-national.php

Ref - https://www.gov.uk/bank-holidays.json